lib/openshift/Consoleclidownload_console_openshift_io.rb

```diff
@@ -0,0 +1,8 @@
+require 'openshift/cluster_resource'
+
+module BushSlicer
+  # @note represents an OpenShift cluster resource about console
+  class ConsoleclidownloadConsoleOpenshiftIo < ClusterResource
```

Comment:
Should the class name be `ConsoleCliDownloadConsoleOpenshiftIo`?

Reply:
@xiaocwan also, the file name should not be capitalized

Reply:
@xiaocwan that's just our Ruby class naming conventions

Reply:
Anyway, I could update.